start_coupled.R

```diff
@@ -297,23 +297,35 @@ start_coupled <- function(path_remind, path_magpie, cfg_rem, cfg_mag, runname, m
         message("Starting subsequent run ", run)
         logfile <- file.path("output", subseq.env$fullrunname, "log.txt")
         if (! file.exists(dirname(logfile))) dir.create(dirname(logfile))
-        if (isTRUE(subseq.env$qos == "auto")) {
+        startnow <- TRUE
+        if (isTRUE(subseq.env$qos %in% c("auto", "multiplayer"))) {
           sq <- system(paste0("squeue -u ", Sys.info()[["user"]], " -o '%q %j' | grep -v ", fullrunname), intern = TRUE)
-          subseq.env$qos <- if (is.null(attr(sq, "status")) && sum(grepl("^priority ", sq)) < 4) "priority" else "short"
+          startnow <- is.null(attr(sq, "status")) && sum(grepl("^priority ", sq)) < 4
+          subseq.env$qos <- if (startnow || subseq.env$qos == "multiplayer") "priority" else "short"
+          if (subseq.env$qos == "auto") startnow <- TRUE
```

Comment:
Lines 305 and 304 should be swapped, shouldn't they? Otherwise `subseq.env$qos == "auto"` can remain `auto` and send to the cluster in line 307.

Reply:
Yes, that is nonsense indeed.